src/adapters/handlers/analyticsDayData.ts

```diff
@@ -1,17 +1,21 @@
 import { IHttpServerComponent } from '@well-known-components/interfaces'
-import { Params } from '../../logic/http/params'
 import { asJSON } from '../../logic/http/response'
+import { AnalyticsTimeframe } from '../../ports/analyticsDayData/types'
+import { getTimestampFromTimeframe } from '../../ports/analyticsDayData/utils'
 import { AppComponents, Context } from '../../types'
 
 export function createAnalyticsDayDataHandler(
-  components: Pick<AppComponents, 'analyticsDayData'>
-): IHttpServerComponent.IRequestHandler<Context<'/analytics/day'>> {
-  const { analyticsDayData } = components
+  components: Pick<AppComponents, 'analyticsData'>
+): IHttpServerComponent.IRequestHandler<Context<'/analytics/:timeframe'>> {
```

Comment:
don't this handler and the one above have the same route? `/analytics/:timeframe`

Reply:
actually i think this handler `src/adapters/handlers/analyticsDayData.ts` is not used at all, we can remove it no?

Reply:
yeah, leftovers 🤦 . Removed!